docs/pipelines/upgrading/upgrading-from-infrastructure-pipelines.md

```diff
@@ -14,7 +14,14 @@ Before you begin the migration process, ensure that you have the following prere
 - AWS Administrator (or similar) permissions in all AWS accounts managed by your `infrastructure-live` repository.
 - A solid understanding of Infrastructure as Code (IAC) fundamentals, or support from Gruntwork.
 
-## Step 1: Update the `bootstrap.yml` workflow
+## Step 1: Ensure Machine Users are configured
```

Comment:
Is this a rebase/merge oops?

Reply:
No, it should be included. I previously updated the `_docs-sources` entry for this file in the web UI before without running the server locally, so the corresponding `docs` entry didn't get updated.

The change here is the result of feedback from you that users need to have their machine users configured before they attempt to bootstrap, as the token required to create the pull request won't be available otherwise.

Reply:
Oh, how weird -- I could _swear_ I reordered those steps already.  Thanks for fixing.